src/platform/forms-system/src/js/review/submit-states/GenericError.jsx

```diff
@@ -54,7 +54,7 @@ export default function GenericError(props) {
       <PreSubmitSection formConfig={formConfig} />
       <Row classNames="form-progress-buttons schemaform-back-buttons vads-u-margin-y--2">
         <Column classNames="small-6 usa-width-one-half medium-6">
-          <a href="/" className="vads-c-action-link--green">
+          <a href="/representative" className="vads-c-action-link--green">
```

Comment:
Obviously we don't want to go back to VA. Once we have a subdoman, this would work for us though.